app/scripts/proactive/config.js

```diff
@@ -62,6 +62,50 @@ define(function () {
             'Script/flow' : [" ", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "powershell", "R"],
             'Script/clean' : [" ", "bash", "cmd", "kubernetes", "docker-compose", "scalaw", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R"]
         },
+        extensions_to_languages: {
+            "sc": "scalaw",
+            "scala": "scalaw",
+            "groovy": "groovy",
+            "js": "javascript",
+            "py" : "python",
+            "rb": "ruby",
+            "ps": "powershell",
+            "r": "R",
+            "sh": "bash",
+            "yaml": "kubernetes",
+            "yml": "docker-compose",
+            "pl": "perl"
+        },
+        languages_to_extensions: {
+            "scalaw" : "sc",
+            "groovy" : "groovy",
+            "javascript" : "js",
+            "python" : "py",
+            "cpython" : "py",
+            "ruby" : "rb",
+            "r" : "r",
+            "bash" : "sh",
+            "cmd" : "sh",
+            "kubernetes" : "yaml",
+            "docker-compose" : "yml",
+            "perl" : "pl",
+            "powershell" : "ps"
+        },
+        languages_content_type: {
+            "scalaw" : "",
+            "groovy" : "",
+            "javascript" : "",
+            "python" : "",
+            "cpython" : "",
+            "ruby" : "",
+            "r" : "",
+            "bash" : "application/x-sh",
+            "cmd" : "",
+            "kubernetes" : "",
+            "docker-compose" : "",
```

Comment:
kubenetes & docker-compose should be text/yaml

Reply:
@fviale shouldn't it be text/x-yaml since it has not yet been accepted into the registered list of MIME types ?